model/src/main/proto/spine_examples/shareaware/wallet/wallet.proto

```diff
@@ -76,6 +76,9 @@ message WalletReplenishment {
 
     // The ID of the wallet to replenish.
     WalletId wallet = 2;
+
+    // The amount of money by which the wallet is replenished.
+    spine.money.Money amount = 3;
```

Comment:
I don't know for sure, but please review this field and the one above for being `required`. They both look like they are.